src/services/DeliverablesService.ts

```diff
@@ -1,315 +1,69 @@
-  // TODO replace with axios
-  return {
-    requestSucceeded: true,
-    deliverable: { ...mockDeliverable },
-  };
-};
+    (response) => ({ deliverable: response?.deliverable })
+  );
 
 const update = async ({ internalComment, reason, status }: UpdateRequest): Promise<Response> => {
   if (status) {
     mockDeliverable.status = status;
   }
 
-  mockDeliverable.reason = reason;
+  mockDeliverable.feedback = reason;
   mockDeliverable.internalComment = internalComment;
   return { requestSucceeded: true };
 };
 
-// TODO will get removed once BE is done
-let mockResponseData: SearchResponseElement[] = [];
-const searchDeliverables = async (
-  fields: string[],
-  organizationId: number,
-  search?: SearchNodePayload,
-  sortOrder?: SearchSortOrder
-): Promise<SearchResponseElement[] | null> => {
-  const params: OptionalSearchRequestPayload = {
-    // TODO confirm prefix when BE is done
-    prefix: 'deliverables',
-    fields,
-    search,
-    // TODO implement pagination when BE is done
-    count: 1000,
-  };
-
-  if (sortOrder) {
-    params.sortOrder = [sortOrder];
-  }
-
-  // TODO change this over once BE is done
-  // return SearchService.search(params);
-  return mockResponseData;
-};
-
-const transformAdminDeliverableElement = (element: SearchResponseElement): SearchResponseDeliverableAdmin => ({
-  category: element.category as DeliverableCategoryType,
-  documentCount: Number(element.documentCount),
-  id: Number(element.id),
-  name: `${element.name}`,
-  project_name: `${element.project_name}`,
-  status: element.category as DeliverableStatusType,
-  // These casts (category, description, type) are typesafe but not runtime safe, if we ever use these cast properties in business logic we should
-  // consider implementing a typeguard to ensure the type is actually the type we're casting to
-  type: element.type as DeliverableTypeType,
-});
-
-const searchDeliverablesForAdmin = async (
-  organizationId: number,
-  search?: SearchNodePayload,
-  sortOrder?: SearchSortOrder
-): Promise<SearchResponseDeliverableAdmin[] | null> => {
-  const result = await searchDeliverables(SEARCH_FIELDS_DELIVERABLES_ADMIN, organizationId, search, sortOrder);
-  if (!result) {
-    return result;
-  }
-
-  return result.map(transformAdminDeliverableElement);
-};
-
-const searchDeliverablesForParticipant = async (
-  organizationId: number,
+const listDeliverables = async (
+  locale: string | null,
+  request?: ListDeliverablesRequestParams,
   search?: SearchNodePayload,
-  sortOrder?: SearchSortOrder
-): Promise<SearchResponseDeliverableParticipant[] | null> => {
-  const result = await searchDeliverables(SEARCH_FIELDS_DELIVERABLES_PARTICIPANT, organizationId, search, sortOrder);
-  if (!result) {
-    return result;
-  }
-
-  return result.map((element) => ({
-    ...transformAdminDeliverableElement(element),
-    description: `${element.description}`,
-  }));
-};
+  searchSortOrder?: SearchSortOrder
+  // TODO sort, search, etc...
```

Comment:
If we will be doing client site sorting, there might be some code [here](https://github.com/terraware/terraware-web/blob/main/src/scenes/PlantingSitesRouter/view/PlantingSitesList.tsx#L114) to extract out as reusable. Handles string vs numeric from the sort order object.
We may need to get number properties localized and in the raw format.

Reply:
Cool, working on that now, will put in another PR